src/components/admin/WorldUserCard/WorldUserCard.tsx

```diff
@@ -1,25 +1,26 @@
 import { faPen, faTrash } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
-import { DISABLED_DUE_TO_MISSING_DESIGN } from "settings";
-
 import { SpaceWithId, UserId, UserWithId } from "types/id";
 
 import { useShowHide } from "hooks/useShowHide";
 
 import { Button } from "../Button";
 import { DeleteAdminModal } from "../DeleteAdminModal";
+import { EditAdminModal } from "../EditAdminModal";
 
 export interface WorldUserCardProps {
   user: UserWithId;
   spaces: SpaceWithId[];
+  worldSpaces: SpaceWithId[];
```

Comment:
What is the difference between `spaces` and `worldSpaces`? I think it might be useful to have some prefix to both of these fields so that `abcdSpaces` and `xyzwSpaces` are more easily distinguished